inc/instance-config.php

```diff
@@ -127,6 +127,7 @@
  */
 $config['country_flags_condensed'] = false;
 $config['user_flag'] = true;
+$config['country_flags'] = true;
```

Comment:
We need `user_flag` to be true for the website to have our custom flags. Let's read the comment for this option, defined in inc/config.php:

```php
// Allow the user choose a /pol/-like user_flag that will be shown in the post. For the user flags, please be aware
// that you will have to disable BOTH country_flags and contry_flags_condensed optimization (at least on a board
// where they are enabled).
```

So it seems the settings `user_flag` and `country_flags` are mutually exclusive. Thus I propose to change [line 114 of inc/api.php](https://github.com/PietroCarrara/leftypol_lainchan/blob/json-flags/inc/api.php#L114) from:

```php

		if (isset($post->body_nomarkup) && $this->config['country_flags']) {
```

to

```php

		if (isset($post->body_nomarkup) && ($this->config['country_flags'] || $this->config['user_flag'])) {
```